scripts/lc-builds/blueos_xl-2018.10.29_omptarget.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I think we can get rid of this build script. The 2018.11.02 one should be sufficient.